s2i-dotnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Source-to-Image
+
+[Source-to-Image (S2I)](https://github.com/openshift/source-to-image)
+is a toolkit and workflow for building reproducible container images
+from source code. S2I produces images by injecting source code into a
+base S2I container image and letting the container prepare that source
```

Comment:
Suggest: S2I injects code into a base S2I container image and lets the container prepare that source code for execution to produce images.
Also please use either s2i or S2I consistently through out the doc.